MAINTAINERS.md

```diff
@@ -0,0 +1,101 @@
+# Kubernetes Project Maintainership
+
+Kubernetes is a big project with many contributors, the project organization is
+a daunting task and the project maintainers are the members that are trusted
+with that task, driving the project, improving the health of the project,
+performing quality control and being responsible over one or more components in
+the project. A Kubernetes maintainer is a contributor who has contributed/is
+contributing to the project regularly and a developer that demonstrated good
+technical judgement. The following is a list of requirements to be a Kubernetes
+maintainer.
+
+Before continuing, it should be noted that we value every contributions, however
+small it may be, even a comment on an issue a valuable contribution to us.
+Maintainership requires time commitment and community trust. You don't have to
+be a maintainer to have a meaningful impact on Kubernetes!
+
+## How to become a maintainer
+
+### 1. Be an active contributor
+
+An active contributor is someone who contributed regularly in the last 2 months
+and is one of the top 20 of contributors for the last 2 months.
+
+For example in the context of merged commit contributions, from September to
+November, the 20th developer has 20 commits.
+
+#### Contribution Types
+
+Contributions are not limited to code. Other contributions include PR reviews,
+discussion participations, test ownership, SIG leadership, etc. At the moment we
+are only able to measure PR contributions. As we improve our project tooling,
+we'll be able to measure and analyze other types of contributions and take those
+into account when evaluating a maintainer application.
+
+### 2. Be experienced in the project
+
+You need to have more than 50 contributions total and project participation for
+more than a year in the Kubernetes organization.
+
+### 3. Find 3 senior maintainer sponsors
+
+As you contribute to Kubernetes, you'll be collaborating with different
+community members. Having the approval of senior project maintainers means that
+they have seen your work and they believe that you'll be a good addition to the
+project. 
```

Comment:
The intent behind the wording is sponsorship. If a senior maintainer sponsors for an applicant, it means that the sponsor saw the applicant in action/development/design/etc. and thinks the applicant would be a good fit